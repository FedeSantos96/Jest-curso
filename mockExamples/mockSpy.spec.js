const { test2 } = require('./funciones');

describe("MockSpy", () => {
    beforeAll(() => {
        jest.spyOn(console, 'info') 
        jest.spyOn(console, 'error')
    })
    it("prueba de exito", () => {
        test2(true);
        expect(console.info).toBeCalled();
        expect(console.info).toBeCalledWith("la info fue correcta");
        expect(test2).toMatchSnapshot();
    })

    it("prueba de falla", () => {
        test2(false);
        expect(console.error).toBeCalled();
    })
})