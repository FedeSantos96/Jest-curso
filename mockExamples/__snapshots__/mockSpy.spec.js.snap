// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MockSpy prueba de exito 1`] = `[Function]`;
